### PR TITLE
wrong path for Assistants/bot-in-a-box

### PR DESCRIPTION
**Pull request by seilorjunior**
# Description

wrong path for the directory.

See the path in the repo https://github.com/Azure/AI-in-a-Box/tree/main/gen-ai/Assistants/bot-in-a-box

### CODE REVIEW

File: scenarios/Assistants/assistant-bot-in-a-box/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The flow of messages is as follows:
 
 ```
 git clone https://github.com/Azure/AI-in-a-Box
-cd gen-ai/assistant-bot-in-a-box
+cd gen-ai/Assistants/bot-in-a-box
```
Comment (brandom-msft) [inline scenarios/Assistants/assistant-bot-in-a-box/README.md:42]: Is this meant for the current repo and directory paths? e.g.
```suggestion
cd scenarios/Assistants/assistant-bot-in-a-box
```

File: scenarios/Assistants/assistant-bot-in-a-box/README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ The flow of messages is as follows:
 
 ```
 git clone https://github.com/Azure/AI-in-a-Box
```
Comment (brandom-msft) [inline scenarios/Assistants/assistant-bot-in-a-box/README.md:41]: repo path is out of date?

**Review by brandom-msft — CHANGES_REQUESTED** [review]

File: scenarios/Assistants/assistant-bot-in-a-box/README.md (hunk 1 above)
Comment (seilorjunior) [reply scenarios/Assistants/assistant-bot-in-a-box/README.md:42]: @brandom-msft just for the current repo that uses this git clone https://github.com/Azure/AI-in-a-Box.

File: scenarios/Assistants/assistant-bot-in-a-box/README.md (hunk 2 above)
Comment (seilorjunior) [reply scenarios/Assistants/assistant-bot-in-a-box/README.md:41]: the repo is fine, but they changed the folder of this repo

**Comment by brandom-msft** [comment]
This code/project looks to be the same as what's in https://github.com/Azure/AI-in-a-Box/tree/main/gen-ai/Assistants/bot-in-a-box, is that correct? If so, is the plan to maintain duplicates? (I think I'm missing some context, hence the previous comments I'd left)

**Review by brandom-msft — APPROVED** [review]
I didn't mean to block this PR, sorry. (I'd still like to understand what the long-term plan is for maintaining this in two repos, but I'll sync w/the team for that)

**Pull request merged**